core/types/transaction.go

```diff
@@ -151,27 +184,27 @@ func (tx *Transaction) ProtoEncode() (*ProtoTransaction, error) {
 	// Other fields are set conditionally depending on tx type.
 	switch tx.Type() {
 	case QuaiTxType:
+		if tx.To() != nil {
```

Comment:
This function was simply rearranged to match the order of elements in QuaiTx for easier verification.